src/NexusMods.DataModel/Loadouts/IngestSteps/RemoveFromLoadout.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NexusMods.DataModel.Loadouts.ApplySteps;
+using NexusMods.Paths;
+
+namespace NexusMods.DataModel.Loadouts.IngestSteps;
+
+/// <summary>
+/// Specifies that the file should be removed from the loadout because something else removed the file from the game folders
+/// </summary>
+public record RemoveFromLoadout : IIngestStep
+{
+    public required AbsolutePath To { get; init; }
```

Comment:
`To` should be renamed to `Path`, since it's the path to the file that should be removed from the loadout.